client/driver/rkt.go

```diff
@@ -78,60 +81,96 @@ func (d *RktDriver) Fingerprint(cfg *config.Config, node *structs.Node) (bool, e
 
 // Run an existing Rkt image.
 func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, error) {
+	// Validate that the config is valid.
 	trust_prefix, ok := task.Config["trust_prefix"]
```

Comment:
The we would need to have instructions on how to pre-populate trusted keys for `rkt`. Docs: https://coreos.com/rkt/docs/0.5.4/commands.html